src/content/docs/en/pages/deploy-journey/deploy-service.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Deploy a service
+description: aaa
+meta_tags: deploy, orchestrator
+namespace: docs_deploy_overview
+menu_namespace: deployMenu
+og_image: ''
+permalink: /documentation/products/deploy/deploy-a-service/
+---
+
+import Button from '~/components/Button.astro'
+
+**Edge Orchestrator** allows you to manage and control edge resources in real time, including the deployment, update, and management of various services and resources.
+
+Azion provides two ways to manage edge nodes, edge services, resources, and credentials: 
+
+- [Azion API](https://api.azion.com/#32f34add-9e9f-4b25-94c9-3cee4c266a23)
+- [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/)
+
+:::note 
+The creation of an edge node happens directly on devices when [Azion Orchestrator Agent]() is installed on them.
+:::
+
+<Button href="/en/documentation/products/deploy/edge-orchestrator/edge-node/" text="go to edge orchestrator reference documentation" variant="secondary"></Button>
+
+---
+
+## Step 1: Create an edge service
+
+To create an edge service, you need to give it an easy-to-remember name.
```

Comment:
To start, you must create an edge service.

Reply:
Alternativa. Deixar o botão nesse caso, para manter alinhado com os outros passos